src/harness/harnessUtils.ts

```diff
@@ -148,7 +148,7 @@ namespace Utils {
         }
     }
 
-    function isNodeOrArray(a: any): boolean {
+    function isNodeOrArray(a: any): a is ts.Node {

```

Comment:
Is this accurate? I'd think this should mention NodeArray, which is not a Node.

Reply:
Will double-check.